erpnext/selling/sales_common.js

```diff
@@ -82,10 +82,6 @@ erpnext.selling.SellingController = erpnext.TransactionController.extend({
 				}
 			});
 		}
-
-		if(this.frm.fields_dict.sales_team && this.frm.fields_dict.sales_team.grid.get_field("sales_person")) {
-			this.frm.set_query("sales_person", "sales_team", erpnext.queries.not_a_group_filter);
-		}
```

Comment:
At present, we do not allow Parent to be selected in the sales-person field as part of the team. This happens since not_a_group_filter() filters out the parent node and Hence, we have removed the code to avoid filtering the parent node.